ocrd_detectron2/segment.py

```diff
@@ -488,7 +488,7 @@ def get_mask(x):
             if self.parameter['debug_img'] != 'none':
                 path = self.workspace.save_image_file(
                     Image.fromarray(visimg.get_image()),
-                    (file_id if isinstance(segment, PageType) else file_id + '_' + segment.id) + '.IMG-DEBUG',
+                    file_id + '_' + region_id + '.IMG-DEBUG',
```

Comment:
Thanks for the proposal. However, that's not how it was meant. A single image with all regions helps get an overview of the various stages (raw and postprocessing).

The actual bug above was the indentation level (region instead of page/table).